Use helm tpl to include diskUtilization value in crawler_args

tw4l opened the pull request:
Related to #787 , where @anjackson pointed out that the way we were including `disk_utilization_threshold` in `crawler_args` wasn't working as intended.

For further reference, see: https://itnext.io/reference-other-values-in-helm-chart-values-file-19d44d9276c7

tw4l committed:
Use helm tpl to include diskUtilization value in crawler_args

ikreymer commented:
Ah, so the way we've done this for other values is to have it be resolved in the configmap.yml instead:
https://github.com/webrecorder/browsertrix-cloud/blob/main/chart/templates/configmap.yaml#L74
```
  CRAWL_ARGS: "{{ .Values.crawler_args }} --workers {{ .Values.crawler_browser_instances | default 1 }} --sizeLimit {{ .Values.crawler_session_size_limit_bytes }} --timeLimit {{ .Values.crawler_session_time_limit_seconds }} --maxPageLimit {{ .Values.max_pages_per_crawl | default 0 }} --healthCheckPort {{ .Values.crawler_liveness_port }}"
```

This may have been an oversight initially - however, the `tpl` command actually gives us more flexibility, as the configmap can then just have:
```
CRAWL_ARGS: "{{ .Values.crawler_args }"
```
and the rest of the values can be overridden in values.yaml. This is more flexible as otherwise this default command can't be changed easily.

To be consistent, I think we should either:
1) Move disk_utilization_threshold to also be configured in the configmap like other properties, no need for tpl
2) Move all of the properties to be in values.yaml, and use tpl.

ikreymer commented:
On the other hand, there are some flags that really shouldn't be changed at all, like --generateWACZ, so maybe it makes sense to have them be in the configmap. Ideally, the full command line shouldn't need to ever be changed, all of the properties that we want to expose, like `--text`, should configurable using individual settings

ikreymer committed:
crawler args cleanup:
- move crawler args command line entirely to configmap
- add required settings like --generateWACZ and --waitOnDone to configmap to not be overridable
- values files can configure individual settings, assembled in configmap
- move disk_utilization_threshold to configmap
- add 'crawler_logging_opts' and 'crawler_extract_full_text' options to values.yaml to more easily set these options

ikreymer commented:
On further thought, refactored to remove the values.yaml `crawler_args` altogether, instead, exposing all of the settings that can be changed as individual options in values.yaml, including adding `crawler_logging_opts` and `crawler_extract_full_text` as new options, while required options, like --generateWACZ and --waitOnDone are set in configmap.yaml as they're not really option for Browsertrix Cloud to work properly.

tw4l reviewed (commented):
Nice changes!
